Source/Scene/parseKTX2.js

```diff
@@ -0,0 +1,225 @@
+import arraySlice from "../Core/arraySlice.js";
+import defined from "../Core/defined.js";
+import PixelFormat from "../Core/PixelFormat.js";
+import RuntimeError from "../Core/RuntimeError.js";
+import VulkanConstants from "../Core/VulkanConstants.js";
+import PixelDatatype from "../Renderer/PixelDatatype.js";
+import { read } from "../ThirdParty/Workers/ktx-parse.modern.js";
```

Comment:
Is `ktx-parse.modern.js` ok to use? Is it compatible with IE11? The reason I ask is because KTX-Parse can also generate `ktx-parse.js` which looks more es5-like, though doesn't seem to work out of the box like `ktx-parse.modern.js`

Reply:
I'm not sure; the [docs](https://github.com/donmccurdy/ktx-parse#api-documentation) don't seem to mention supported browsers. Maybe cc @mramato here?

Did we want to support KTX2 in IE11?

Reply:
No action needed since we plan to drop support for IE11 soon. Which reminded me, we should add `ktx-parse` to the list of npm packages: https://github.com/CesiumGS/cesium/issues/9473#issuecomment-866257585